test/utils.js

```diff
@@ -0,0 +1,70 @@
+import { watchFile, readdirSync, statSync } from 'fs';
+import path from 'path';
+
+/**
+ * Ignored directories.
+ */
+
+var ignore = [ 'node_modules', '.git' ];
+
+/**
+ * Taken from: https://github.com/mochajs/mocha/blob/8a37e016e5629b46709891740c74d5f0b69929e0/lib/utils.js#L168
+ *
+ * Watch the given `files` for changes
+ * and invoke `fn(file)` on modification.
+ *
+ * @param {Array} files the array of files
+ * @param {Function} fn the function to call on modification
+ */
+exports.watch = function( files, fn ) {
+	var options = { interval: 100 };
+	files.forEach( function( file ) {
+		watchFile( file, options, ( curr, prev ) => {
+			if ( prev.mtime < curr.mtime ) {
+				fn( file );
+			}
+		} );
+	} );
+};
+
+/**
+ * Taken from: https://github.com/mochajs/mocha/blob/8a37e016e5629b46709891740c74d5f0b69929e0/lib/utils.js
+ *
+ * Ignored files.
+ *
+ * @param {string} fp file path
+ * @return {boolean} true if should ignore
+ */
+function ignored( fp ) {
+	return ! ~ignore.indexOf( fp );
```

Comment:
Ah, now I'm seeing that these are pulled from Mocha. Can we not just import directly from the package?